mbed_flasher/common.py

```diff
@@ -119,29 +119,24 @@ def retry(logger, func, func_args, retries=DEFAULT_RETRY_AMOUNT, conditions=None
     :param conditions: conditions on when to retry
     :return: latest return code
```

Comment:
docstring update? no `return_value` anymore

Reply:
On success case it will return normally, line 127.

Reply:
okay